v2/contributing/getting-started.md

```diff
@@ -59,4 +59,8 @@ You will now be able to push new branches to your fork and create [pull requests
     # Run all tests
     php core/scripts/run-tests.sh --verbose --concurrency 4 --url http://d8.d.dev "commerce"
     
-Replace `http://d8.d.dev` with the url to your installation.
\ No newline at end of file
+Replace `http://d8.d.dev` with the url to your installation.
+
+To be able to run individual PHPUnit test (using command line or your IDE), [this article](https://www.drupal.org/node/2116263#skipped-tests) will give more information on how to run specific tests, and setup variables.
```

Comment:
Should we have a subheading on "Running tests"? I have a feeling we could end up adding a bit to this.